bitnami/common/templates/_affinities.tpl

```diff
@@ -12,7 +12,7 @@ preferredDuringSchedulingIgnoredDuringExecution:
           operator: In
           values:
             {{- range .values }}
-            - {{ . }}
+            - {{ quote . }}
```

Comment:
You can also use a pipe:
```suggestion
            - {{ . | quote }}

```

Reply:
Ahh that was my initial impulse, but I switch between Jinja and Go templates so much I never recall which is which. `{{ . | quote }}` looked "too Jinja-ish" and the Helm docs suggested `{{ quote . }}`.

I'll change it around to be consistent with the other charts and bump the appversion.